docs/contracts/PortfolioBridge.md

```diff
@@ -263,13 +264,14 @@ function setGasForDestinationLzReceive(uint256 _gas) external
 
 #### addToken
 
-Adds the given token to the portfoBrige. PortfoBrigeSub the list will be bigger as they could be from
```

Comment:
One more change here - PortfoBrigeSub --> PortfolioBridgeSub

Reply:
done

Reply:
Sorry but still one more "d" missing in line 267 the